README.rst

```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
 
    ./manage.py lms forum_delete_course_from_mongodb all --dry-run
 
+Building MongoDB Indexes
+------------------------
+
+To optimize MongoDB query performance, it is crucial to create database indexes. The command will create or update indexes and skip them if they already exist.
+
+To create or update MongoDB indexes, execute the following command:
+
+    ./manage.py lms forum_create_mongodb_indexes
+
+Search indicies
+---------------
+
+Based on your search backend i.e Elasticsearch or Meilisearch, the commands will populate search indexes.
+
+Initialize Forum Indices
+-------------------------
+
+The `initialize_forum_indices` command used to initialize Elasticsearch indices for the forum. It allows you to force the creation of new indices even if they already exist.
+
+    ./manage.py lms initialize_forum_indicies
+
+Pass `--force` flag to force the creation of new indices.
+
+Rebuild Forum Indices
+----------------------
+
+The `rebuild_forum_indices` command is used to rebuild Elasticsearch indices from scratch. This command allows you to specify the batch size for processing documents and extra minutes for catch-up.
+
+To run the command, use the following syntax:
+
+    ./manage.py lms rebuild_forum_indices --batch_size <size> --extra_catchup_minutes <minutes>
```

Comment:
Let's not mention the batch_size and extra_catchup_minutes options. They are not necessary, since they have good defaults.